front-end/src/screens/Home/LatestDiscussion/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { useLatestDiscussionPostsQuery } from '../../../generated/graphql';
+import Discussions from '../../../components/Listings/DiscussionsListing';
```

Comment:
Same here, let's call it the same as the component, I think it's more understandable